process-engine/skeleton
# ✨Electron Skeleton

## What did you change?

~~**⏰ waits on https://github.com/essential-projects/datasource_adapter_localstorage/pull/4**~~

I added a variant of the skeleton that uses the localstorage-datasource-adapter instead of the postgres-one. This variant works without a dedicated database, so it can be used in a standalone electron application

## How can others test the changes?

- enter the `process-engine-electron`-folder
- run `npm install`
- run `npm run build`
- link https://github.com/essential-projects/datasource_adapter_localstorage/pull/4 into that folder
- run `npm start`
- see how you now have a process-engine instance without a dedicated database

## PR-Checklist

Please check the boxes in this list after submitting your PR:
